packages/client/hmi-client/src/components/tabs/Tab.vue

```diff
@@ -96,12 +96,14 @@ section {
 .content {
 	background-color: white;
 	visibility: hidden;
+	z-index: 0;
```

Comment:
Please add `isolation: isolate;` on the `.outer-container` to make sure those z-index conflicts with nothing else.